tests/public_api_tests/test_backwards_compatible.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
isn't it better to structure the tests around "topic" rather than having a catch-all bucket for backwards comp like this? I think for someone writing code in the future it's easier if it's all in topic based buckets?